engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/types/EspTypeUtils.scala

```diff
@@ -17,18 +17,18 @@ object EspTypeUtils {
 
   def clazzDefinition(clazz: Class[_])
                      (implicit settings: ClassExtractionSettings): ClazzDefinition =
-    ClazzDefinition(Typed.typedClass(clazz), extractPublicMethodsAndFields(clazz))
+    ClazzDefinition(Typed.typedClass(clazz), extractPublicMethodsAndFields(clazz, staticMethodsAndFields = false), extractPublicMethodsAndFields(clazz, staticMethodsAndFields = true))
 
-  private def extractPublicMethodsAndFields(clazz: Class[_])
-                                          (implicit settings: ClassExtractionSettings): Map[String, List[MethodInfo]] = {
+  private def extractPublicMethodsAndFields(clazz: Class[_], staticMethodsAndFields: Boolean)
+                                           (implicit settings: ClassExtractionSettings): Map[String, List[MethodInfo]] = {
     val membersPredicate = settings.visibleMembersPredicate(clazz)
```

Comment:
@mproch @arkadius I have a question for you. How would it be different if we implemented this logic using settings visibleMembersPredicate ? It looks as if such a feature is already implemented there and only a specific predicate is missing